router.go

```diff
@@ -142,6 +142,10 @@ type Router struct {
 	// Custom OPTIONS handlers take priority over automatic replies.
 	HandleOPTIONS bool
 
+	// Configurable http.Handler which is called after modifying the automatic reply
+	// to OPTIONS requests. If HandleOptions is false this won't be used
+	OPTIONSPostHandler http.Handler
```

Comment:
The name is not consistent with the existing interface, see [`Router.HandleMethodNotAllowed`](https://godoc.org/github.com/julienschmidt/httprouter#Router.HandleMethodNotAllowed) and  [`Router.MethodNotAllowed`](https://godoc.org/github.com/julienschmidt/httprouter#Router.MethodNotAllowed). It should thus be called [`Router.OPTIONS`](https://godoc.org/github.com/julienschmidt/httprouter#Router.OPTIONS), but that exists already as a method.

 I think we should either allow to reuse that and allow to register a default handler somehow (e.g. path `*`), or use a somehow more consistent name. Maybe `DefaultOPTIONS`. I don't know what is less confusing.